modules/gapi/include/opencv2/gapi/streaming.hpp

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2020 Intel Corporation.
+//
+// This software and the related documents are Intel copyrighted materials,
+// and your use of them is governed by the express license under which they
+// were provided to you ("License"). Unless the License provides otherwise,
+// you may not use, modify, copy, publish, distribute, disclose or transmit
+// this software or the related documents without Intel's prior written
+// permission.
+//
+// This software and the related documents are provided as is, with no
+// express or implied warranties, other than those that are expressly
+// stated in the License.
+//
+
+#ifndef OPENCV_GAPI_STREAMING_STREAMING_HPP
```

Comment:
Does it make sense to move already existing operations to there ? 
Or maybe remove this header altogether

Reply:
Why it is not
```
OPENCV_GAPI_STREAMING_HPP
```
?

Reply:
> Does it make sense to move already existing operations to there ?

I'd rather put this stuff to `streaming/format.hpp` or something like that _AND_ make sure this file is included by default.

Reply:
Why do you mean  "by default"?

Reply:
Moved to `streaming/format.hpp` not sure if everything is done correctly

Reply:
> Why do you mean "by default"?

It means for user it would be enough to only include `opencv2/gapi.hpp` to have all this functionality. This file could be included in `gstreaming.hpp` (which I believe is already included in `gcomputation.hpp`)

Reply:
There is a cyclic dependency problem:
```
// Include these files here to avoid cyclic dependency between
// Desync & GKernel & GComputation & GStreamingCompiled.
#include <opencv2/gapi/streaming/desync.hpp>
```
Added directly in `gapi.hpp`

Reply:
Not sure where it comes from, but can't you include `streaming/format` in `gapi.hpp` the same way?

Reply:
It's there

Reply:
Ok I see, thanks!